AssemblyUnhollower/Contexts/MethodRewriteContext.cs

```diff
@@ -47,7 +49,7 @@ public MethodRewriteContext(TypeRewriteContext declaringType, MethodDefinition o
             var newMethod = new MethodDefinition("", AdjustAttributes(originalMethod.Attributes, originalMethod.Name == "Finalize"), declaringType.AssemblyContext.Imports.Void);
             NewMethod = newMethod;
 
-            if (originalMethod.CustomAttributes.Any(x => x.AttributeType.FullName == typeof(ExtensionAttribute).FullName))
+            if (HasExtensionAttribute = originalMethod.CustomAttributes.Any(x => x.AttributeType.FullName == typeof(ExtensionAttribute).FullName))
```

Comment:
I suggest moving the assignment out of the `if` to keep code a bit more legible.
i.e.

```cs
HasExtensionAttribute = originalMethod.CustomAttributes.Any(x => x.AttributeType.FullName == typeof(ExtensionAttribute).FullName);

if(HasExtensionAttribute)
...
```